Word-document/Merge-documents-in-new-page/.NET/Merge-documents-in-new-page/Program.cs

```diff
@@ -10,20 +10,20 @@ class Program
     {
         static void Main(string[] args)
         {
-            using (FileStream sourceStreamPath = new FileStream(Path.GetFullPath(@"../../../Data/SourceDocument.docx"), FileMode.Open, FileAccess.Read, FileShare.ReadWrite))
+            using (FileStream sourceStreamPath = new FileStream(Path.GetFullPath(@"Data/SourceDocument.docx"), FileMode.Open, FileAccess.Read, FileShare.ReadWrite))
             {
                 //Opens an source document from file system through constructor of WordDocument class.
                 using (WordDocument sourceDocument = new WordDocument(sourceStreamPath, FormatType.Automatic))
                 {
-                    using (FileStream destinationStreamPath = new FileStream(Path.GetFullPath(@"../../../Data/DestinationDocument.docx"), FileMode.Open, FileAccess.Read, FileShare.ReadWrite))
+                    using (FileStream destinationStreamPath = new FileStream(@"Data/DestinationDocument.docx", FileMode.Open, FileAccess.Read, FileShare.ReadWrite))
```

Comment:
Why the Path.GetFullPath is removed here

Reply:
Addressed the concerns